TgeeT/Questioner_v2
# signup validation and response feedback fix

#### What does this PR do?

This PR is meant to add validations and fixing bug of user duplicating data
#### Description of Task to be completed?

add validation and edge cases

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone this repository [here](https://github.com/TgeeT/Questioner_v2.git) 
2. Create a virtual environment and activate it.
3. git checkout to bg-signup-feedback-163402830  branch
4. Install requirements with 'pip install -r requirements.txt'
5. Run the application and with post man, post at the endpoint 'api/v2/user'

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/163402830
